analysis_idacs.tex

```diff
@@ -0,0 +1,8 @@
+\textbf{I-201} Retrieve PSF cutouts from coadd images (e.g., using the Notebook Aspect of the RSP) to derive parameters of PSF models.
+Use these parameters to train machine-learning-based algorithms for galaxy classification using GPU resources at IDACs.
+
+\textbf{I-202}: Find galaxy clusters in LSST data using Rubin APIs and notebooks for exploratory analysis.
```

Comment:
For a NERSC-based case the IDAC functionality is high-performance compute and co-located datasets. Focus on that. Don't say "download the data" b/c that is not something an individual user should be doing.

Reply:
Confirmed resolved.